group/edwards25519/suite.go

```diff
@@ -1,24 +1,29 @@
 package edwards25519
 
 import (
-	"crypto/cipher"
 	"crypto/sha256"
-	"crypto/sha512"
 	"hash"
 	"io"
 	"reflect"
 
 	"github.com/dedis/crypto"
 	"github.com/dedis/crypto/cipher/sha3"
-	"github.com/dedis/crypto/util/random"
 )
 
+// NOTE: This structure is not removed so this brings the questions of naming and
+// consistency:
+// + what is a suite ? why would we call it a suite in this package and not something else
+// in curve25519 etc ?
+// + how do I know which method to implement ? For example. we'll want to add
+// ed448 to the our list, so we'll need to lookup here to see which methods
+// ed448 is supposed to implement to be sure one can ecchange edwards25519 and
+// edwards448 without any issues.
+// The Suite interface solves these issues by defining the common set of method
+// that be implemented (exactly as this `suiteed25519` suggests).
```

Comment:
I +1 your suggestion for `curveEd25519` or `groupEd25519` or whatever, but don't keep the `suite` in here.